runtime/bcverify/vrfyhelp.c

```diff
@@ -579,6 +579,19 @@ isClassCompatible(J9BytecodeVerificationData *verifyData, UDATA sourceClass, UDA
 	if (NULL != verifyData->vmStruct->currentException) {
 		return (IDATA) FALSE;
 	}
+	
+	if (J9ROMCLASS_IS_HIDDEN(verifyData->romClass)) {	
+		J9UTF8* className = J9ROMCLASS_CLASSNAME(verifyData->romClass);
+		if (sourceLength == J9UTF8_LENGTH(className)
+			&& (0 == strncmp(sourceName, J9UTF8_DATA(className), sourceLength))
+		) {
+			/* isRAMClassCompatible() cannot find ram class of a hidden class, we are testing if 
+			 * the source class is the subclass of target class. We can use superclass of source class instead here. */
```

Comment:
Formatting nitpick - comment closer `*/` should line up with the other comment line starters
```suggestion
			/* isRAMClassCompatible() cannot find ram class of a hidden class, we are testing if 
			 * the source class is the subclass of target class. We can use superclass of source class instead here. 
			 */
```

Reply:
Fixed